README.md

```diff
@@ -0,0 +1,44 @@
+# Repository for the open-source pinball game red-roses
+
+[![Software License][ico-license]](LICENSE.md)

```

Comment:
Lets go with the MIT license, we are good guys!

Reply:
Oh, sorry. I just opened an issue about this. Will close